src/routeTree.gen.ts

```diff
@@ -21,6 +22,11 @@ const IndexRoute = IndexImport.update({
   getParentRoute: () => rootRoute,
 } as any)
 
+const VendorIndexRoute = VendorIndexImport.update({
+  path: '/vendor/',
+  getParentRoute: () => rootRoute,
+} as any)
```

Comment:
Is there a good reason to use `as any` here or is this just a temporary fix because we haven't integrate to the backend?

Reply:
This file is auto-generated by tanstack router, let's leave it as it is